change/@fluentui-web-components-2021-01-12-10-32-22-chore-update-webpack5.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
@dzearing This doesn't impact any of the distributed packages (or it shouldn't) - should this really bump the version?

Reply:
Well, likely it should not really impact anything if the package doesn't include any output anything from webpack. i was patching just to be safe.

Reply:
put this as "none" for web-components

Reply:
Saw that, which is why you have my approval 👍